.github/workflows/lint.yml

```diff
@@ -0,0 +1,20 @@
+---
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+        with:
+          submodules: 'true'
```

Comment:
This repo doesn't have a `.gitmodules` file, so there are no submodules, and we don't need this setting.